loops-arrays-objects.js

```diff
@@ -113,6 +156,34 @@ const students = [
     {name: 'Aicha', city: 'Utrecht', zipCode: '3514', neighborhood: null},
     {name: 'Karima', city: 'Utrecht', zipCode: '3531', neighborhood: null},
 ];
+console.log(students);
+for (let i = 0; i < students.length; i++) {
+    // console.log(students[i].zipCode);
+    switch (true) {
```

Comment:
bij switch statement schrijf je tussen de haken datgene wat je wilt vergelijken met de cases die je hebt gedeclareerd.  In dit geval schrijf je dus die zip code  uit het object (zoals je bij console.log erboven al testte) na het switch key word.  En hierdoor kan je bij alle cases die === vergelijking weglaten en gewoon alleen het nummer zetten. Zie hieronder: 
for (let i = 0; i < students.length; i++) {
    switch (students[i].zipCode) {
        case "3513":
            students[i].neighborhood = "Pijlsweerd";
            break;
        case "3514":
            students[i].neighborhood = "Vogelenbuurt";
            break;  etc